ocsp.go

```diff
@@ -315,59 +346,48 @@ func validateOCSP(ocspRes *ocsp.Response) *ocspStatus {
 	}
 }
 
-func retryOCSPCacheServer(
+func isTestUnknownStatus() bool {
+	return strings.EqualFold(os.Getenv(ocspTestInjectUnknownStatusEnv), "true")
+}
+
+func checkOCSPCacheServer(
 	client clientInterface,
 	req requestFunc,
 	ocspServerHost *url.URL,
-	totalTimeout time.Duration,
 	httpTimeout time.Duration) (
```

Comment:
If my understanding is correct, this is rest timeout rather than http timeout. The http timeout is specified in client.